index.html

```diff
@@ -131,9 +143,14 @@ <h4>
             "!promises-guide#a-new-promise">a new promise</a>.
             </li>
             <li>If none of <var>data</var>'s members <a>title</a>, <a>text</a>,
-            or <a>url</a> are present, reject <var>p</var> with
+            <a>url</a> or <a>files</a> are present, reject <var>p</var> with
             <a>TypeError</a>, and abort these steps.
             </li>
+            <li>If the implementation does not support file sharing and none of
```

Comment:
This does not seem sustainable; imagine adding another optional member, the permutations.

I would rather just delete this, and add a parenthetical above, like this:

"If none of data's members title, text, or url are present, reject p with url or (if the implementation supports file sharing) files are present"